SPECS/safeclib/safeclib.spec

```
@@ -0,0 +1,72 @@
+Summary:        C11 Annex K functions
+Name:           libsafec
+Version:        3.7.1
+Release:        1%{?dist}
+License:        MIT
+URL:            https://github.com/rurban/safeclib
+Source0:        https://github.com/rurban/safeclib/releases/download/v3.7.1/safeclib-%{version}.tar.xz
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  gcc
+BuildRequires:  make
+%if %{with_check}
+BuildRequires:  tzdata
+%endif
+
+%description
+Extension library providing C11 Annex K functions
+
+%package -n libsafec-devel
+Summary:        Development packages for libsafec
+Requires:       libsafec = %{version}-%{release}
+
+%description -n libsafec-devel
+Development files for libsafec
+
+%package -n libsafec-check
+Summary:        Tools to detect use of unsafe libc APIs
+
+%description -n libsafec-check
+Tools to detect use of unsafe libc APIs
+
+%prep
+%setup -q -n safeclib-%{version}
+
+%build
+autoreconf -Wall --install
+%configure --disable-static --enable-debug --enable-strmax=0x8000
+%make_build
+
+%if %{with_check}
+%check
+export TZ=Europe/London
+make check %{?_smp_mflags}
+%endif
+
+%install
+%make_install
+install -d %{buildroot}%{_mandir}/man1/ %{buildroot}%{_mandir}/man3/
+install -D doc/man/man1/* %{buildroot}%{_mandir}/man1/
+install -D doc/man/man3/* %{buildroot}%{_mandir}/man3/
+find %{buildroot} -type f -name "*.la" -delete -print
+find %{buildroot} -type f -name "*.pc" -delete -print
+
+%files -n libsafec
+%license COPYING
+%{_libdir}/libsafec.so.*
+
+%files -n libsafec-devel
+%license COPYING
+%{_libdir}/libsafec.so
+%{_includedir}/safeclib
+%{_mandir}/man3/*
+
+%files -n libsafec-check
+%license COPYING
+%{_bindir}/check_for_unsafe_apis
+%{_mandir}/man1/*
```

Same thing for the `-n` switch in the `%files` sections:

```suggestion
%files
%license COPYING
%{_libdir}/libsafec.so.*

%files devel
%license COPYING
%{_libdir}/libsafec.so
%{_includedir}/safeclib
%{_mandir}/man3/*

%files check
%license COPYING
%{_bindir}/check_for_unsafe_apis
%{_mandir}/man1/*
```